### PR TITLE
Improve autophp - remove termwind

### PR DESCRIPTION
**Pull request by f-lombardo**
See https://github.com/theodo-group/LLPhant/pull/167#issuecomment-2218243753

### CODE REVIEW

File: src/Experimental/Agent/AutoPHP.php [hunk 1]
```diff
@@ -34,22 +32,23 @@ public function __construct(
...
+    public function run(int $maxIteration = 10): string
     {
         $this->outputAgent->renderTitle('🐘 AutoPHP 🐘', '🎯 Objective: '.$this->objective, $this->verbose);
         $this->creationTaskAgent->createTasks($this->objective, $this->tools);
         $this->outputAgent->printTasks($this->verbose, $this->taskManager->tasks);
         $currentTask = $this->prioritizationTaskAgent->prioritizeTask($this->objective);
         $iteration = 1;
         while ($currentTask instanceof Task && $maxIteration >= $iteration) {
+            $this->outputAgent->render('Iteration '.$iteration, false);
+            $this->outputAgent->render('Iteration '.$iteration, false);
```
Comment (MaximeThoonsen) [inline src/Experimental/Agent/AutoPHP.php:51]: @f-lombardo small mistake I guess ?

**Comment by MaximeThoonsen** [comment]
I small change to do otherwise 👍️ @f-lombardo

File: src/Experimental/Agent/AutoPHP.php (hunk 1 above)
Comment (f-lombardo) [reply src/Experimental/Agent/AutoPHP.php]: Yes, thank you!

**Comment by MaximeThoonsen** [comment]
perfect thanks.

**Pull request merged**